spec/spec_helper.rb

```diff
@@ -74,8 +74,13 @@
                 end
   end
 
-  config.after(:each) do
-    ActiveJob::Base.queue_adapter = :test
+  config.around(:each) do |example|
```

Comment:
Nemuze ti tohle prebit naky jiny hook?

Reply:
Tak vypada ze dobry - https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/around-hooks
Mysel by to nekdo definovat "vys".